src/utils/aiprdescription/fetchGithubAPIData.ts

```diff
@@ -0,0 +1,23 @@
+interface Commit {
+  commit: {
+    message: string;
+  };
+}
+
+export const getPRDiff = async (url: string) => {
+  const response = await fetch(
+url,
+    { headers: { Accept: "application/vnd.github.v3.diff" } },
+  );
+  const diff = await response.text();
```

Comment:
Need error handling here. What happens if the response errors here?

Reply:
Will do.